mono/metadata/threads.h

```diff
@@ -54,7 +54,7 @@ MONO_API void mono_threads_request_thread_dump (void);
 
 MONO_API mono_bool mono_thread_is_foreign (MonoThread *thread);
 
-extern MONO_API void mono_thread_detach_if_exiting (void);
+extern MONO_API int32_t mono_thread_detach_if_exiting (void);
```

Comment:
I fixed it at https://github.com/mono/mono/pull/3074/files#diff-f1a9287559d3aa620b058619f47f00c9L1085